app/code/community/Drip/Connect/Helper/Data.php

```diff
@@ -12,16 +12,16 @@ class Drip_Connect_Helper_Data extends Mage_Core_Helper_Abstract
      *
      * @return bool
      */
-    public function isModuleActive()
+    public function isModuleActive($store = null)
     {
-        if (!empty(Mage::app()->getRequest()->getParam('store'))) {
-            return (bool)Mage::getStoreConfig(
-                'dripconnect_general/module_settings/is_enabled',
-                Mage::app()->getRequest()->getParam('store')
-            );
+        // TODO: Refactor this method away by using the config object at all the call sites.
```

Comment:
Incidentally, this refactor is here: https://github.com/DripEmail/magento-m1-extension/pull/23

For once this PR gets merged.